guides/17_Migration_to_3_0.md

```diff
@@ -162,6 +162,51 @@ handle them please remove.
 - StateMachineTransitionFailed
 - AudioDeviceSwitched
 
+## AudioVideo events
+
+The following `AudioVideo` events have been removed in v3.x, these events redirect data from `metricDidReceive`.
+
+- videoSendHealthDidChange
+- videoSendBandwidthDidChange
+- videoReceiveBandwidthDidChange
+
+In 3.x you can still replecate these events funtionality through `metricDidReceive`.
+
+
+```js
+const observer = {
+  oldSendBandwidth: 0,
+  oldRecvBandwidth: 0,
+  metricsDidReceive: (clientMetricReport) => {
+    const metricReport = clientMetricReport.getObservableMetrics();
+
+    const videoUpstreamPacketPerSecond = metricReport.videoPacketSentPerSecond;
+    const videoUpstreamBitrate = metricReport.videoUpstreamBitrate;
+    const availableSendBandwidth = metricReport.availableOutgoingBitrate;
+    const availableRecvBandwidth = metricReport.availableIncomingBitrate;
+    const nackCountPerSecond = metricReport.nackCountReceivedPerSecond;
```

Comment:
nit: This could be simplified as:

```js
const {
  videoPacketSentPerSecond,
  videoUpstreamBitrate,
  availableOutgoingBitrate,
  availableIncomingBitrate,
  audioSpeakerDelayMs,
} = metricReport;
```

I think "Outgoing" and "Incoming" should be fine.

Reply:
Updated. I kept the send and recv as I actually needed to convert to kbs

Reply:
Then you can do this:
```js
const {
  videoPacketSentPerSecond,
  videoUpstreamBitrate,
  availableOutgoingBitrate: availableSendBandwidth,
  availableIncomingBitrate: availableRecvBandwidth,
  audioSpeakerDelayMs,
} = metricReport;
```

You don't have to change it. Current code looks good to me.